Vector/Views/Authentication/AuthInputsView.m

```diff
@@ -419,6 +459,100 @@ - (void)updateAuthSessionWithCompletedStages:(NSArray *)completedStages didUpdat
     }
 }
 
+- (BOOL)setExternalRegistrationParameters:(NSDictionary *)registrationParameters
```

Comment:
ok. I see.
I mixed [AuthInputsView setExternalRegistrationParameters] and [AuthIViewController setExternalRegistrationParameters].